install-deps.sh

```diff
@@ -545,4 +568,4 @@ if $for_make_check; then
     type git > /dev/null || (echo "Dashboard uses git to pull dependencies." ; false)
```

Comment:
And we should probably rewrite this... Dashboard used `git` long time ago since some NPM dependency was pinned to a github repo, instead to an npmjs published package, but not anymore. However, `git` is used by the `run-make-check` script to obtain the SHA of the current branch.

Reply:
OK, good to know.